framework/Models/PostProcessors/DataClassifier.py

```diff
@@ -93,133 +93,93 @@ def _handleInput(self, paramInput):
         func = child.findFirst('Function')
         funcType = func.parameterValues['type']
         funcName = func.value.strip()
-        self.mapping[child.parameterValues['name']] = (funcType, funcName)
+        varName = child.parameterValues['name']
+        if varName in self.mapping.keys():
+          self.raiseAnError(IOError, "The variable {} name is duplicated in the XML input".format(varName))
+        self.mapping[varName] = (funcType, funcName)
       elif child.getName() == 'label':
         self.label = child.value.strip()
 
-  def inputToInternal(self, currentInput):
+  def identifyInputs(self, currentInput):
     """
-      Method to convert a list of input objects into the internal format that is
-      understandable by this pp.
-      @ In, currentInput, list, a list of DataObjects
-      @ Out, newInput, list, list of converted data
+      Method to identify the inputs for classifier and target, respectively
```

Comment:
can you expand the description here? What "identify" means?

Reply:
updated